lms/djangoapps/discussion/views.py

```diff
@@ -307,11 +307,36 @@ def forum_form_discussion(request, course_key):
             'corrected_text': query_params['corrected_text'],
         })
     else:
+        discussions_mfe_enabled_for_everyone = ENABLE_DISCUSSIONS_MFE_FOR_EVERYONE.is_enabled(course_key)
```

Comment:
This code block is repeated 3 times, it can be moved to the util function. 

```python
discussions_mfe_enabled_for_everyone = ENABLE_DISCUSSIONS_MFE_FOR_EVERYONE.is_enabled(course_key)
    if discussions_mfe_enabled_for_everyone and (not is_privileged_user(course_key, request.user)):
```

Reply:
Other than that implementation looks good.